src/gui/pedalboard/cables/Cable.cpp

```diff
@@ -102,7 +102,16 @@ void Cable::paint (Graphics& g)
 
         endPortLocation = CableViewPortLocationHelper::getPortLocation ({ endEditor, connectionInfo.endPort, true }).toFloat();
         endColour = endEditor->getColour();
-        levelDB = connectionInfo.endProc->getInputLevelDB (connectionInfo.endPort);
+        auto updatedLevelDb = jlimit (floorDB, 0.0f, connectionInfo.endProc->getInputLevelDB (connectionInfo.endPort));
+        auto levelDifference = std::abs (updatedLevelDb) - std::abs (levelDB);
```

Comment:
```suggestion
        auto levelDifference = std::abs (updatedLevelDb - levelDB);
```

Reply:
Good catch, added this in the latest changes.